index.html

```diff
@@ -27,114 +27,163 @@
+            <button id="btn_next" class="btn btn-sm">Next</button>
+          </div>
+
+        </div>  
+
+        <div id="tab_2" class="box d-none">
+       
+          <select id="input_condition" name="condition" class="input ">
+            <option value="after">
+              After of
+            </option>
+            <option value="before">
+              Before of
+            </option>
+            <option value="equals">
+              Equals to
+            </option>
+          </select>
+          
+          <input type="date" name="date" id="input_date" class="calendary_filter__date_search input" />
+
+          <div class="p-relative">
+            <button id="goback" class="btn--back p-absolute mt--6">
+              <img src="./src/assets/arrow.png" style="margin-top: 3px" width="15px" height="15px" alt=""/>
+            </button>
+          </div>
 
+        
+        
+          <div class="div mt-3">
+            <button id="redirectToSearch" class="btn btn-sm">Ir</button>
+            <button id="copySearch" class="btn btn--secondary btn-sm">Copiar</button>
+          </div>
+        </div>
+        <datalist id="repositories_history">
+          <!-- TODO: ADD repositories history on this datalist -->
+        </datalist>
+        <datalist id="users_history">
+          <!-- TODO: ADD users history on this datalist -->
+        </datalist>
+        
+        
+    </form>
+  </div>
+</div>
+  </main>
+  
   <script type="module">
-    import { getFromLSOrDefaultValue, validateForm, createSearchFilter, getFormValues } from './src/js/utils.js';
-
-    const form = document.forms['calendary_filter'];
-    const formData = getFromLSOrDefaultValue('formData', getFormValues(form));
-    const linkRedirect = document.getElementById('redirectToSearch');
-    const buttonCopy = form['copySearch'];
-
-    const fillFormWithFormData = (formData, form) => {
-      if (!form) return;
-
-      for (const [key, data] of Object.entries(formData)) {
-        try {
-          form[key].value = data;
-        } catch (error) {
-          debugger;
-          console.error(error);
-        }
-      }
-    }
+   import { getFromLSOrDefaultValue, validateForm, createSearchFilter, getFormValues } from './src/js/utils.js';
+
+/*
+* STEPS FORM 
+*/
+const btnNext =document.getElementById('btn_next');
+const goBack =document.getElementById('goback');
+const tab1 = document.getElementById('tab_1');
+const tab2 = document.getElementById('tab_2');
+
+btnNext.addEventListener('click',function(evt){
+  evt.preventDefault();
+  tab2.classList.remove('d-none');
+  tab1.classList.add('d-none');
+});
 
-    window.onload = () => {
-      const date = new Date();
-      const fullDate = `${date.getFullYear()}-${date.getMonth()}-${date.getDay() >= 10 ? date.getDate() : '0' + date.getDate() }`;
+goBack.addEventListener('click',function(evt){
+  evt.preventDefault();
+  tab1.classList.remove('d-none');
+  tab2.classList.add('d-none');
+});
```

Comment:
What do you think about to use some like this?:

```javascript
  const switchTab = (evt) {
     evt.preventDefault();
     tab1.toggle('d-none');
     tab2.toggle('d-none');
  }

  btnNext.addEventListener('click', switchTab);
  
  goBack.addEventListener('click', switchTab);
```

It's just a proposal since I don't know if it can cause some error or something. Let me know what you think